charts/moco/README.md

```diff
@@ -58,6 +58,100 @@ Users must manually upgrade the CRD if there is a change in the CRD used by MOCO
 
 https://helm.sh/docs/chart_best_practices/custom_resource_definitions/#install-a-crd-declaration-before-using-the-resource
 
+## Migrate to v1
+
+Chart version v1 has breaking changes.
+The `.metadata.name` of the resource generated by Chart is changed.
+
+e.g.
+
+* `{{ template "moco.fullname" . }}-foo-resources` -> `moco-foo-resources`
+
+Related Issue: [cybozu-go/moco#426](https://github.com/cybozu-go/moco/issues/426)
+
+If you are using a release name other than `moco`, you need to migrate.
```

Comment:
Could you explain the impact on existing MySQLClusters?
For example, is there a downtime of the existing MySQLCluster?

Reply:
https://github.com/cybozu-go/moco/pull/440/commits/e1f590b3ac69942851de37f902919a24474b20af

updated.
Is my understanding correct?